clients/request.go

```diff
@@ -209,7 +210,7 @@ func PutRequest(url string, body []byte, ctx context.Context) (string, error) {
 	}
 
 	if body != nil {
-		req.Header.Set(ContentType, ContentJson)
+		req.Header.Set(ContentType, ContentTypeJson)
```

Comment:
Check `Context` first to make sure ContentType isn't already set, then default to JSON. Same as `PostRequest()` above.

Reply:
Okay. So we're setting the content type to JSON in both the request header and the context, correct?